p2p/security/noise/transport.go

```diff
@@ -7,13 +7,19 @@ import (
 	"github.com/libp2p/go-libp2p/core/canonicallog"
 	"github.com/libp2p/go-libp2p/core/crypto"
 	"github.com/libp2p/go-libp2p/core/peer"
+	"github.com/libp2p/go-libp2p/core/protocol"
 	"github.com/libp2p/go-libp2p/core/sec"
+	"github.com/libp2p/go-libp2p/p2p/security/noise/pb"
 
 	manet "github.com/multiformats/go-multiaddr/net"
 )
 
 // ID is the protocol ID for noise
-const ID = "/noise"
+const (
+	ID                 = "/noise"
+	MAX_EXTENSION_SIZE = 2048
+	MAX_PROTO_NUM      = 100
```

Comment:
Please stick with Go variable naming conventions.

Reply:
My understanding is tht go constant name convention is all capital separated by underscores, do we have some different convention we adopt in this project?

Reply:
I don't think that's the convention. Look at https://pkg.go.dev/net/http#pkg-constants for example.

Reply:
I see, changed the names. thanks!